### DISCUSSION TITLE
Trivy KBOM for RKE clusters

### DISCUSSION THREAD

**Discussion by itaysk** [rancher/rke2#5329] [General]
Hi there, [Trivy](https://github.com/aquasecurity/trivy) is an open source vulnerability scanner that recently introduced a feature where it can scan Kubernetes clusters core components for vulnerabilities. (for example, you are running kubelet v1.2.3 which is vulnerable to CVE321 so please update to kubelet v1.2.4) [Here](https://thenewstack.io/akin-to-sbom-trivy-adds-kbom-vulnerability-scanning-to-k8s/) is an overview of this.
Currently this works for upstream components, and we are looking to start working with vendors for supporting other k8s distros so that open source Trivy users can start scanning for example RKE clusters as well.
If possible we'd love to connect with someone about the packaging and patching strategy of Rancher.
Thanks!

**Comment by brandond** [comment]
Were you looking at RKE specifically, or RKE2 and K3s?

**Reply by itaysk** [reply]
RKE2

**Comment by cwayne18** [comment]
Hi Itay,
We'd be happy to sync up on this with our product folks if you're up for it.  Please reach out to chris.wayne@suse.com and I'll try and get something set up.  Thanks so much for thinking of us

**Reply by itaysk** [reply]
Thanks! send you an email
